src/askForNumber.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+'use strict';
+
+const readline = require('readline');
+const classNames = require('classnames');
+
+const { isUniqueNumber } = require('./isUniqueNumber');
+const { constants } = require('./utils/constants');
+
+const getMessageForUser = classNames;
+const getPostfix = classNames;
+
+const { MAX_LENGTH } = constants;
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+let attempts = 1;
+
+function askForNumber(question, numberToGuess) {
+  terminal.question(question, (userInput) => {
+    let countOfBulls = 0;
+    let countOfCows = 0;
+
+    const isCorrectNumberLength = userInput.length !== MAX_LENGTH;
+    const hasUniqueNumbers = !isUniqueNumber(userInput) && userInput.length === MAX_LENGTH;
+
+    const INCORRECT_LENGTH_MESSAGE = `Current length is ${userInput.length}, you should input only ${MAX_LENGTH} digits: `;
+    const NON_UNIQUE_NUMBER = 'Your number contains non-unique digits, enter only unique: ';
+    const NUMBERS_ONLY = 'You must use only numbers: ';
```

Comment:
It is better to define this constants outside the component so that it is not created every time a new function calling is performed.